CHANGELOG/v1.7.0-beta.1.md

```diff
@@ -0,0 +1,411 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
+
+* Enable kubeadm feature gates mutation
+
+<details>
+<summary>More details about the release</summary>
+
+:warning: **BETA RELEASE NOTES** :warning: 
+
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.29.x
+- Workload Cluster: v1.24.x -> v1.29.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.6.0
+## :chart_with_upwards_trend: Overview
+- 298 new commits merged
+- 4 breaking changes :warning:
+- 11 feature additions ✨
+- 36 bugs fixed 🐛
+
+## :memo: Proposals
+- MachinePool: Update MachinePool Machines proposal with diagrams (#9664)
+
+## :warning: Breaking Changes
+- API: Remove v1alpha4 API Version (#9939)
```

Comment:
This got reverted later in https://github.com/kubernetes-sigs/cluster-api/pull/10147
I think it's confusing if it still ends up here as a breaking change